libs/api/domains/finance/src/lib/api-domains-finance.resolver.ts

```diff
@@ -1,16 +1,6 @@
 import { Query, Resolver, Args } from '@nestjs/graphql'
-import { GetFinancialOverviewInput } from './dto/getOverview.input'
-import { GetCustomerRecordsInput } from './dto/getCustomerRecords.input'
-import { GetDocumentsListInput } from './dto/getDocumentsList.input'
-import { GetFinanceDocumentInput } from './dto/getFinanceDocument.input'
-import { GetAnnualStatusDocumentInput } from './dto/getAnnualStatusDocument.input'
-import { ForbiddenException, UseGuards } from '@nestjs/common'
+import { ForbiddenException, Inject, UseGuards } from '@nestjs/common'
 import graphqlTypeJson from 'graphql-type-json'
-import { CustomerChargeType } from './models/customerChargeType.model'
-import { FinanceDocumentModel } from './models/financeDocument.model'
-import { CustomerTapsControlModel } from './models/customerTapsControl.model'
-import { DocumentsListModel } from './models/documentsList.model'
-import { CustomerRecords } from './models/customerRecords.model'
```

Comment:
Nice ... like that you're cleaning the order of imports!

Would be awesome to hook up a formatter like [isort](https://github.com/PyCQA/isort) to structure the imports like:
```
3rd party

2nd party (libs)

1st party
```
in order to keep this consistent across the repo